### PR TITLE
`page.create` dialog: fix single template value

### LINKED ISSUE #3644: When debug is set to false and there is only one blueprint available, Kirby uses the default blueprint instead of the defined blueprint
**Describe the bug**  
If a section only allows one template and the `debug` option is set to false in config, the default template is used instead of the defined template in Kirby 3.6.0-alpha.4 , because `$template` is then set to null in `dialogs.php` instead of being assigned the template name.

**To Reproduce**  
1. In a 3.6-alpha-4 StarterKit, set `'debug'  => false` in config
2. Open the Panel and create a new `photography` or `notes` subpage.
3. Notice how it is created with the default template instead of the `album` or `note` templates.

**Expected behavior**  
The defined blueprint should be used.

**Kirby Version**  
3.6.0-alpha.4

### PR DESCRIPTION
**Pull request by distantnative**
### Fixes
- Page create dialog: correct template used when not in `debug` mode and only one template is allowed

## Breaking changes

None

## Related issues/ideas

- Fixes #3644

## Ready?

- [ ] Unit tests for fixed bug/feature
- [ ] In-code documentation (wherever needed)
- [ ] CI checks pass

## When merging

- [ ] Add to [website docs release checklist](https://github.com/getkirby/getkirby.com/pulls) (if needed)
- [ ] Add changes to release notes draft in Notion

### CODE REVIEW

**Comment by distantnative** [comment]
Can't figure out where `Page` is coming from
Screen Shot 2021-09-22 at 15 21 39

**Comment by afbora** [comment]
`Page` is default template name and should be like that:

````php
$this->assertSame('Page', $props['value']['template']);
````

File: config/areas/site/dialogs.php [hunk 1]
```diff
@@ -263,7 +263,7 @@
             $blueprints = $view->blueprints($section);
 
             // the pre-selected template
-            $template = null;
+            $template = $blueprints[0]['name']  ?? $blueprints[0]['value'] ?? null;
```
Comment (afbora) [inline config/areas/site/dialogs.php]:
```suggestion
            $template = $blueprints[0]['name'] ?? $blueprints[0]['value'] ?? null;
```

**Comment by distantnative** [comment]
@bastianallgeier why are we using `name` here as well and not just `value`? https://github.com/getkirby/kirby/blob/develop/src/Panel/Field.php#L209

**Comment by bastianallgeier** [comment]
> @bastianallgeier why are we using `name` here as well and not just `value`? https://github.com/getkirby/kirby/blob/develop/src/Panel/Field.php#L209

It's the same as in the line above for the title. The blueprints already have a title and name prop and this way you don't have to convert the blueprints array to options before passing it to the field.

**Review by bastianallgeier — APPROVED** [review]

**Pull request merged**